### PR TITLE
Solpio/UI/auth

### PR DESCRIPTION
**Pull request by Solpio**

### CODE REVIEW

File: admin-app/src/constants/colors.ts [hunk 1]
```diff
@@ -12,6 +12,9 @@ export enum ColorsDark {
   widgetBackGround = "#23262d",
   activeNavLinkBackground = "#30333c",
   widgetBorderColor = "#272a31",
+  authTitleText = "#f7b03e",
+  authInputLabel = "#7e7d7d",
+  authInputFilled = "#ffffff",
 }
```
Comment (kurkul608) [inline admin-app/src/constants/colors.ts:17]: authTitleText looks like dateText

File: admin-app/src/global.ts [hunk 2]
```diff
@@ -2,6 +2,9 @@ import { createGlobalStyle } from "styled-components";
 import { color } from "./constants/colors";
 
 export default createGlobalStyle`
+     html {
+      font-size: 10px;
+     }
```
Comment (kurkul608) [inline admin-app/src/global.ts:13]: add font-size for tablet, and mobile width

File: admin-app/src/pages/auth-page/index.tsx [hunk 3]
```diff
@@ -1,9 +1,10 @@
 import React from "react";
+import {AuthForm} from "../../shared/auth-form";
 
 export const AuthPage = () => {
     return (
         <>
-            <div>Auth Page</div>
+          <AuthForm></AuthForm>
```
Comment (kurkul608) [inline admin-app/src/pages/auth-page/index.tsx]: use <... /> view

File: admin-app/src/pages/auth-page/index.tsx [hunk 4]
```diff
@@ -1,9 +1,10 @@
 import React from "react";
+import {AuthForm} from "../../shared/auth-form";
```
Comment (kurkul608) [inline admin-app/src/pages/auth-page/index.tsx]: why you don't use prettier?

File: admin-app/src/shared/auth-form/components/auth-form-input/index.tsx [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+    <>
+      <AuthInputWrapper>
+        <AuthInput
+          onChange={(e) => {
+            setValue(e.target.value);
+            onChange(e.target.value);
+          }}
+          id={"login"}
+          autoComplete={"off"}
+        ></AuthInput>
```
Comment (kurkul608) [inline admin-app/src/shared/auth-form/components/auth-form-input/index.tsx]: <../>

File: admin-app/src/shared/auth-form/components/auth-form-submit/index.tsx [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+import { Submit } from "./styeld";
+
+interface ISubmit {
+  onSubmit: any;
+}
+
+export const AuthFormSubmit = ({ onSubmit }: ISubmit) => {
+  return (
+    <>
+      <Submit
```
Comment (kurkul608) [inline admin-app/src/shared/auth-form/components/auth-form-submit/index.tsx]: button must be reusable and located in shared/components

File: admin-app/src/shared/auth-form/components/auth-form-input/index.tsx [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+  onChange: any;
+}
+
+export const AuthFormInput = ({ onChange }: IAuthFormProps) => {
+  const [value, setValue] = useState("");
+
+  return (
+    <>
+      <AuthInputWrapper>
+        <AuthInput
```
Comment (kurkul608) [inline admin-app/src/shared/auth-form/components/auth-form-input/index.tsx]: input must be reusable and located in shared/components

File: admin-app/src/global.ts (hunk 2 above)
Comment (Solpio) [reply admin-app/src/global.ts:13]: i don't give a fuck about scale of size

**Review by kurkul608 — APPROVED** [review]

**Pull request merged**